docker-compose.yml

```diff
@@ -93,31 +98,28 @@ services:
       - RESOURCEGROUP
       - SECRET_SA_ACCOUNT_NAME
       - STORAGE_ACCOUNT_DOMAIN
-
-      # override
       - ARO_ADOPT_BY_HIVE=true
       - ARO_CHECKOUT_PATH=/app
       - ARO_INSTALL_VIA_HIVE=true
       - HIVE_KUBE_CONFIG_PATH=/app/secrets/aks.kubeconfig
       - KUBECONFIG=/app/secrets/aks.kubeconfig
       - RP_MODE=development
-    expose:
-      - "8443"
     ports:
-      - "127.0.0.1:8443:8443"
+      - "8443:8443"
     healthcheck:
-      test: ["CMD", "curl", "-k", "http://localhost:8443/healthz"]
-      interval: 30s
+      test: ["CMD", "curl", "-k", "-sS", "https://localhost:8443/healthz/ready"]
+      interval: 90s
       timeout: 30s
-      retries: 3
+      retries: 5
     restart: on-failure:3
 
   portal:
     image: ${LOCAL_ARO_RP_IMAGE}:${VERSION}
     container_name: aro-portal
+    network_mode: host
     depends_on:
       rp:
-        condition: service_healthy
+        condition: service_started
```

Comment:
what new information required this change? I'd imagine that we want the RP up and serving traffic by the time we start Portal - no?

Reply:
Agreed, updated accordingly